workflow/scripts/grabMetrics.py

```diff
@@ -29,12 +30,45 @@ def generateQCMetrics(args):
     # Generate PeakFileQCSummary.txt in Peaks Directory#
     pred_metrics = PeakFileQC(pred_metrics, args.macs_peaks, args.outdir)
     # Appends Percentage Counts in Promoters into PeakFileQCSummary.txt
-    pred_metrics = NeighborhoodFileQC(
-        pred_metrics, args.neighborhood_outdir, args.outdir, "DHS"
-    )
-    pred_metrics = NeighborhoodFileQC(
-        pred_metrics, args.neighborhood_outdir, args.outdir, "H3K27ac"
-    )
+    if (
+        len(
+            glob.glob(
+                os.path.join(
+                    args.neighborhood_outdir, "Enhancers.DHS.*CountReads.bedgraph"
+                )
+            )
+        )
+        > 0
+    ):
```

Comment:
Is it just me or does the formatting here look bad?

Reply:
yeah it looks strange but I reformatted it using black so not sure what's going on. I could try again.

Reply:
i think black is working as intended. It's probably because there's too much in the if statement s.t it's splitting it that way. Feel free to change it if you wish though